lessons/06-apis-debugging/passportExample/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
The example apps are awesome -- maybe point to them somewhere in the readme/inclass/homework and give the high-level explanation of what they're doing, and/or write a quick README for each sample which explains what it does and how to run it?